modules/ROOT/pages/apikit-set-header-task.adoc

```diff
@@ -3,41 +3,81 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In this procedure, you add header information to the outbound response that complies with the WSDL. Header information that complies with the WSDL document is called APIUsageInformation.
+Using APIkit you can set headers to your application's response based on the WSDL declarations.
 
-. In Studio, open the tshirt2.wsdl in `src/main/wsdl` and scroll to APIUsageInformation element, which is the element expected by the contract. Copy the name of the element to the clipboard.
-. On the canvas, select Transform message in the `OrderTshirt:/TshirtService/TshirtServicePort/api-config` flow, and in the properties editor, click Add New Target:
-+
-image::apikit-for-soap-3005f.png[height=440,width=700]
-+
-. Select Property instead of Variable from the drop-down.
-. In Variable Name, paste the contents of the clipboard, and add *soap.* as a prefix. or type *soap.APIUsageInformation*.
-+
-The complete variable name looks like this:
-+
-soap.APIUsageInformation
-+
-. Click OK.
-. Double-click apiCallsRemaining: _Integer_.
+To follow this example, you must have downloaded the `tshirt2.wsdl` API definition, and created a Mule application based on it. +
+See xref:apikit-soap-prerequisites-task.adoc[Prerequisites for Using APIkit for Soap] and xref:apikit-soap-project-task.adoc[Create an APIkit for SOAP Project] for more information.
+
+In this example, you add the `APIUsageInformation` header to the response of your application:
+
+. In Studio, open the `tshirt2.wsdl` in `src/main/resources/api` and scroll to `APIUsageInformation` element, which is the element expected by the contract.
+
+. On the canvas, select the Transform Message component on the `OrderTshirt:/soapkit-config` flow.
+. In the existing Dataweave code:
```

Comment:
Same comment as with the other doc - what is the action you are doing here -entering code?